src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -30712,20 +30712,567 @@ public interface I1
 
     static I1() {}
 }
+interface I2
+{
+    I2();
+}
+
 ";
             var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll,
                                                  parseOptions: TestOptions.Regular);
 
-            // From https://github.com/dotnet/csharplang/blob/master/meetings/2017/LDM-2017-05-17.md:
-            // We allow initializers for static fields, but not an explicit static constructor. Can think about adding later.
             compilation1.VerifyDiagnostics(
-                // (4,5): error CS0526: Interfaces cannot contain constructors
+                // (4,5): error CS0526: Interfaces cannot contain instance constructors
                 //     I1(){}
                 Diagnostic(ErrorCode.ERR_InterfacesCantContainConstructors, "I1").WithLocation(4, 5),
-                // (6,12): error CS0526: Interfaces cannot contain constructors
+                // (10,5): error CS0501: 'I2.I2()' must declare a body because it is not marked abstract, extern, or partial
```

Comment:
Nit: could we suppress this error if CS0526 is reported? This error doesn't matter: whether or not the interface constructor has a body is irrelevant. It's illegal either way.

Reply:
I do not believe it is worth it.

---
In reply to: [260428025](https://github.com/dotnet/roslyn/pull/33625#discussion_r260428025) [](ancestors = 260428025)